src/routes/safe/components/Balances/SendModal/screens/ReviewTx/index.tsx

```diff
@@ -44,9 +47,13 @@ const ReviewTx = ({ closeSnackbar, enqueueSnackbar, onClose, onPrev, tx }) => {
   const [gasCosts, setGasCosts] = useState('< 0.001')
   const [data, setData] = useState('')
 
-  const txToken = tokens.find((token) => token.address === tx.token)
-  const isSendingETH = txToken.address === ETH_ADDRESS
-  const txRecipient = isSendingETH ? tx.recipientAddress : txToken.address
+  const txToken = React.useMemo(() => tokens.find((token) => token.address === tx.token), [tokens, tx.token])
```

Comment:
tx.token is an object, will equality check for this case? if tx changes then it will be re-run I think

Reply:
I have to double-check this, it's a string:

<kdb><img src="https://user-images.githubusercontent.com/3315606/92145783-1f850780-edef-11ea-97b4-13ae4fdff579.png" /></kbd>